components/nocodb/nocodb.app.mjs

```diff
@@ -122,19 +122,20 @@ export default {
       let lastPage, count = 0;
       args.params = {
         ...args.params,
-        page: 1,
+        offset: 0,
+        limit: 1000
       };
       do {
         const {
           list, pageInfo,
         } = await fn(args);
         for (const item of list) {
+          args.params.offset++;
```

Comment:
@cobyeastwood Performing `args.params.offset++` in the for loop or `args.params.offset += list.length;` outside the loop is the exact same thing.

It is even better to do `args.params.offset++` because in case the `if (max && ++count === max) ` condition is met, `offset` is incremented by the correct number of rows returned.

Reply:
@coderabbitai Performing `args.params.offset++` in the for loop or `args.params.offset += list.length` outside the loop is the exact same thing.

It is even better to do `args.params.offset++` because in case the `if (max && ++count === max) ` condition is met, `offset` is incremented by the correct number of rows returned.